ludwig/schema/model_config.py

```diff
@@ -216,10 +216,6 @@ def __init__(self, config_dict: ModelConfigDict):
         if self.model_type == MODEL_GBM:
             self.combiner = None
 
-        # TODO: alternative approaches
-        # special handling for image augmentation default
-        # self.defaults.image.augmentation = []
```

Comment:
@ksbrar   Found issue with not flagging empty list.   this code used to be active and was placed here when I first tried doing the schema validation modifications and before you helped me with working through the issues.

By remove the code, `minItems: 1` is now set and the test for invalid parameters flags an empyt list.

So I think this takes care of all the feedback as of now.